source/slang/slang-ir-inst-defs.h

```diff
@@ -497,6 +497,10 @@ INST(GroupMemoryBarrierWithGroupSync, GroupMemoryBarrierWithGroupSync, 0, 0)
 // GPU_FOREACH loop of the form 
 INST(GpuForeach, gpuForeach, 3, 0)
 
+// Wrapper for OptiX intrinsics used to load and store ray payload data using
+// a pointer represented by two payload registers.
+INST(GetPRD, getPRD, 0, 0)
```

Comment:
Might be good to give this operation a more distinctive opcode name/mnemonic. While `getPRD()` is the name given to the function right now, I might argue that `getOptiXRayPayloadPtr` is more accurate to what the semantics are here.

Reply:
Good call, should now be updated in the latest commit